### PR TITLE
HTTP/2 blog post

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jonmv** [comment]
💯

File: _posts/2021-06-29-http2.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+allow [Transport Layer Security (TLS)](https://datatracker.ietf.org/doc/html/rfc5246) version 1.2 or newer.
+Less secure features and weaker cipher suites from *TLSv1.2* are not allowed. Most browser vendors have taken an aggressive
+stance on security by supporting HTTP/2 over TLS only.
+
+## Adding HTTP/2 support to Vespa
+[Jetty 9.4](https://www.eclipse.org/jetty/) is the HTTP stack powering the Vespa Container.
+As *Jetty 9.4* has supported HTTP/2 for many years already, implementing HTTP/2 support for Vespa Container was more or
+less straightforward, with high performance out of the box.
+Like for most web browsers, we decided to adopt HTTP/2 over TLS only, although for slightly different reasons:
+- Firstly, Vespa at Verizon Media and [Vespa Cloud](https://vespa.ai) is required to implement _all_ Vespa protocols—both
```
Comment (bratseth) [inline _posts/2021-06-29-http2.md:39]: url should be cloud.vespa.ai

File: _posts/2021-06-29-http2.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+![HTTP/1.1 vs HTTP/2 throughput](/assets/2021-06-29-http2/http1-vs-http2.png)
+
+# Simple, yet effective feeding over HTTP/2 against /document/v1
+
+[`/document/v1`](https://docs.vespa.ai/en/reference/document-v1-api-reference.html) is a REST-ified
+HTTP API which exposes the [Vespa Document API](https://docs.vespa.ai/en/document-api-guide.html) to
+the outside of the application's Java containers. The design of this API is simple: each feed operation
+is modelled as a single HTTP request, and its result as a single HTTP response. While it was previously
+not possible to achieve comparable throughput using `/document/v1` to what the 
+[Vespa HTTP Client](https://docs.vespa.ai/en/vespa-http-client.html) could achieve against the (undocumented)
```
Comment (bratseth) [inline _posts/2021-06-29-http2.md:74]: "internal" instead of "(undocumented)"?

Comment (jonmv) [reply _posts/2021-06-29-http2.md:74]: Yes.

File: _posts/2021-06-29-http2.md (hunk 1 above)
Comment (jonmv) [reply _posts/2021-06-29-http2.md:39]: Of course.

Comment (jonmv) [reply _posts/2021-06-29-http2.md:39]:
```suggestion
- Firstly, Vespa at Verizon Media and [Vespa Cloud](https://cloud.vespa.ai) is required to implement _all_ Vespa protocols—both
```

File: _posts/2021-06-29-http2.md (hunk 2 above)
Comment (jonmv) [reply _posts/2021-06-29-http2.md:74]:
```suggestion
[Vespa HTTP Client](https://docs.vespa.ai/en/vespa-http-client.html) could achieve against the internal,
```

**Comment by jonmv** [comment]
🥳

**Pull request merged**